### ISSUE TITLE
Moving Users Between Vesting Orgs

### ISSUE THREAD

**Issue by cdemarco** [harvard-lil/perma#834]
I know we've already talked about this somewhat, but I'm working with a Registrar User at BU who is having a hard time.  In order to change a Vesting User's Org, she has to remove the User from "Default Vesting Org" and then re-add them to "BU Law Review" (for example).  Why can't it behave the way it does for us where I can select a new Vesting Org from the dropdown and click save changes"?

**Comment by phillipsm** [comment]
Right. @abziegler and I just looked at this. It's super easy to shift vesting users between vesting orgs if you're a registry member. But, if you're a registrar member, it's a lot more work -- you have to kick the vesting member out of the org (likely and then add them).

tldr; the "move vesting users between vesting orgs" action for registrar members should be simplified. It should look like it does for registry members.

**Comment by anniejocain** [comment]
I'm logged in as Registrar Member, I hit 'edit/remove' on any vesting user and am given a drop down of vesting organizations to choose from. Just like a registry member.

@cdemarco @phillipsm @abziegler can you check again? Am I missing something?

**Comment by abziegler** [comment]
Registrar user goal: move vesting user A from vesting org X to vesting org Y.

Current Flow:
* Find vesting user A
* Click remove 
* Confirm remove from vesting org X
* Click "add user"
* Enter user A's email 
* Select vesting org Y from dropdown
* Click add button

Desired Flow:
* Find vesting user A
* Vesting org X is indicated
* Select vesting org Y from dropdown
* Click "move to new vesting org" button

**Comment by anniejocain** [comment]
Flow is already as we want it, closing.
